master/internal/restore.go

```diff
@@ -119,6 +119,7 @@ func (e *experiment) restoreTrial(
 	if trialID != nil {
 		t.processID(*trialID)
 	}
+	t.processOperations(ops)
```

Comment:
q: why moving this function here?

Reply:
ah, this was just so the ops would be there in the restore call, just incase it needed them. it doesn't matter though since it doesnt use it.